### DISCUSSION TITLE
whisper have mypy stubs library for type annotation?

### DISCUSSION THREAD

**Discussion by ZeyaKim** [openai/whisper#1999] [Q&A] [answered]
I think It exist, but I can't find anywhere even stackoverflow.

how can I use type annotation in whisper?

![image](https://github.com/openai/whisper/assets/118428615/598545b1-5843-4661-906d-95bbe046c2a5)

**Comment by phineas-pta** [comment, marked as answer]
> whisper have mypy stubs library for type annotation?

no
